lib/bloc_record/selection.rb

```diff
@@ -33,6 +38,65 @@ def find_by(attribute, value)
     init_object_from_row(row)
   end
 
+  def find_each(*options = {})
+    if options.nil?
```

Comment:
Unless someone calls `find_each(nil)`, this will not be true. There is also the case that someone may provide one option and not the other (either they only want the first 25 so passing a start seems redundant, or they want to start at some place and grab everything after that).